samples/nrf9160/modem_shell/src/link/link_api.c

```diff
@@ -386,194 +353,115 @@ void link_api_coneval_read_for_shell(void)
 }
 
 /* ****************************************************************************/
-#define XMONITOR_RESP_REG_STATUS_VALID 1
-#define XMONITOR_RESP_FULL_NAME_VALID 2
-#define XMONITOR_RESP_SHORT_NAME_VALID 4
-#define XMONITOR_RESP_PLMN_VALID 8
-#define XMONITOR_RESP_BAND_VALID 16
-#define XMONITOR_RESP_CELL_ID_VALID 32
-#define XMONITOR_RESP_RSRP_VALID 64
-#define XMONITOR_RESP_SNR_VALID 128
-
-#define OPERATOR_FULL_NAME_STR_MAX_LEN 124
-#define OPERATOR_SHORT_NAME_STR_MAX_LEN 64
-#define OPERATOR_CELL_ID_STR_MAX_LEN 32
-#define OPERATOR_PLMN_STR_MAX_LEN 32
+
+#define OP_FULL_NAME_STR_MAX_LEN 128
```

Comment:
This has increased. Was it on purpose?

Reply:
yes, increased by 4 bytes. 128 is used elsewhere as well for this. Anyways, it is safe even though if modem would return even larger.